Added flags for 1-QDF and full month fraction

samhollenbach opened the pull request:
The flags should be working now. I tested observer 412 with all 4 flag combinations, and got working but different plots. The flags only change the outcome if ADF_flag set to "QDF" and Month flag set to "FULL". Otherwise, the original calculations are used.

amunozj commented:
Hi Sam! great work with the implementation.  I made some changes.  Can you please review them and let me know if they work so that I can merge?

samhollenbach commented:
This commit seems stable right now. I am trying to run it to see if the issue with data not being assigned (~line 298 in ADF_SSN.py) is fixed. So far it is running well, but I cant remember exactly which observer made it crash. Otherwise, should be good on the other changes. I changed how the flags are assigned, added more command line options, and let the user set default values in the config file. The default values are only there so the program will not crash if no flags are set, but we can change this if you want the user to always define their flags through the command line or at the top of the main script.
